src/coverlet.template/coverlet.template.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
```

Comment:
Are you sure this change is correct? I believe the template targets netstandard2.0 because of the references that get copied over into the instrumented assembly with Mono.Cecil. At least for S.P.CoreLib we are removing the netstandard references again afterwards.

Reply:
Yes. On .NET Framework builds, the façade assemblies are not guaranteed to be available.